javascript/javascript1/week2/week2.js

```diff
@@ -0,0 +1,159 @@
+//fullname function
+function getFullname(fullname1, fullname2) {
+    return fullname1 + " " + fullname2
+}
+console.log(getFullname("Benjamin", "Hughes"))
+
+
+//Formal fullname
+function getFullname(fullname1, fullname2, useFormalName = false, gender = "male") {
+    if (useFormalName) {
+        if (gender === "male") {
+
+            return "Lord" + fullname1 + " " + fullname2
+        } else {
+            return "Miss" + fullname1 + " " + fullname2
+        }
+    } else {
+        return fullname1 + " " + fullname2
+    }
+
+}
+
+
+//Event application
+const day = ["sunday", "monday", "tuesday", "wendsday", "thursday", "friday", "saterday"]
+
+function getEventWeekday(today, eventDay) {
+    let eventDay_ = eventDay % 7;
+
+    let index = -1;
+    for (let i = 0; i < day.length; i++) {
+        if (today === day[i]) {
+            index = i;
+            break;
+        }
+    }
+
+    if (index === -1) {
+        return undefined;
+    }
+
+    return "The event will take place in " + day[(index + eventDay_) % 7];
```

Comment:
you can also directly add it here without defining the variable because it's the only place you use it. I mean
`day[(index + eventDay_) % 7 => day[(index+eventDay_)%7]`